pkg/selinux/setroubleshoot-view.jsx

```diff
@@ -149,7 +149,7 @@ class SELinuxEventDetails extends React.Component {
             }
 
             return (
-                <StackItem key={itm.analysisId + (itm.ifText || "") + (itm.doText || "")}>
+                <StackItem key={`stack-item-${itmIdx}`}>
```

Comment:
This is wrong. The key must be a representation of the *intrinsic* contents of the element. The list index is extrinsic -- specifying that is not useful, as the very point is that React can see elements which are already rendered even if the list changes. Think about it like this: If that would work, then we wouldn't need keys in the first place, as react can easily enumerate the list itself.

So the key needs to be extended with whatever disambiguates the two entries. In your screenshot above, the two solutions look exactly identical, so that feels like a bug in setroubleshoot. We could work around this, keep a set of `ifText+doText` and filter out duplicates ourselves. But this would also affect CLI users.

Reply:
So lets look into SELinux:

```
fileIf you think this is caused by a badly mislabeled machine.touch /.autorelabel; reboot
```

```
{
    "ifText": "If you think this is caused by a badly mislabeled machine.",
    "thenText": "you need to fully relabel.",
    "doText": "touch /.autorelabel; reboot",
    "analysisId": "file",
    "fixable": true,
    "reportBug": false
}
```

This appears twice.

Likely caused by:
```
{
    "ifText": "If you want to fix the label. \n/run/foo default label should be default_t.",
    "thenText": "you can run restorecon. The access attempt may have been stopped due to insufficient permissions to access a parent directory in which case try to change the following command accordingly.",
    "doText": "# /sbin/restorecon -v /run/foo",
    "analysisId": "restorecon",
    "fixable": true,
    "reportBug": false
}
```

Reply:
So we get our access control alerts from Dbus:
```
[root@fedora-40-127-0-0-2-2201 ~]# busctl call org.fedoraproject.Setroubleshootd /org/fedoraproject/Setroubleshootd org.fedoraproject.SetroubleshootdIface get_all_alerts
a(ssi) 1 "0f2d23cf-b1fc-4bea-abcb-816e449357b3" "SELinux is preventing systemd-fstab-g from getattr access on the directory /run/foo." 46
```

And 
```
busctl call org.fedoraproject.Setroubleshootd /org/fedoraproject/Setroubleshootd org.fedoraproject.SetroubleshootdIface get_alert "s" "0f2d23cf-b1fc-4bea-abcb-816e449357b3"
```

To retrieve the details.